template.tpl

```diff
@@ -153,12 +153,12 @@ ___TEMPLATE_PARAMETERS___
               {
                 "type": "REGEX",
                 "args": [
-                  "^([3-9]|[1-9][0-9])\\..+$"
+                  "^([4]|[0-9][0-9])\\..+$"
```

Comment:
This can probably be just `"^4\\..+$"`, the extra bits are to allow up to version 99 without accepting leading zeroes or version 1/2. This instead accepts 4 or 00 - 99, including 01, 02, 03, 05+ which we probably explicitly don't want if we're doubling down on a Tag per major version. Also we don't need square brackets for a single character. :)

Also good to note that the [referenced page](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracker-setup/hosting-the-javascript-tracker/third-party-cdn-hosting/) in the docs hasn't been updated to mention v4. (Or remove v2 since we apparently no longer support that, TIL).

Reply:
Cool, have updated regex and got a [PR](https://github.com/snowplow/documentation/pull/1076) for the docs update.